content/en/docs/concepts/workloads/pods/podpreset.md

```diff
@@ -51,9 +51,9 @@ Pods, Kubernetes modifies the Pod Spec. For changes to `Env`, `EnvFrom`, and
 the Pod; for changes to `Volume`, Kubernetes modifies the Pod Spec.
 
 {{< note >}}
-A Pod Preset is capable of modifying the `.spec.containers` field in a
-Pod spec when appropriate. *No* resource definition from the Pod Preset will be
-applied to the `initContainers` field.
+A Pod Preset is capable of modifying the `.spec.containers` field and, 
```

Comment:
I recommend splitting this into two sentences / keeping this as 2 sentences. That's because  I think the two shorter sentences are both easier to read in English and easier to localize.

Reply:
Good point, I have refactored the note in the following update.